README.md

```diff
@@ -59,6 +69,8 @@ Know your OS
 
 ## Communication
 Chat: [Gitter Channel](https://gitter.im/fossasia/meilix) | [Get an Invite](http://fossasia-slack.herokuapp.com/)
+Please join our mailing list to discuss questions regarding the project: https://groups.google.com/forum/#!forum/meilix
+Scrum report for the repository will be send to the address: meilix@googlegroups.com
```

Comment:
Good to add this. Why should one send a scrum report?

Reply:
As codeheat is going on and most of the new developers asked to send the scrum report of the week, so I mentioned that.